setup.py

```diff
@@ -146,7 +150,10 @@ def get_long_description():
                   'run = avocado.plugins.run:Run',
                   'sysinfo = avocado.plugins.sysinfo:SysInfo',
                   'plugins = avocado.plugins.plugins:Plugins',
-                  ]
+                  ],
+              'avocado.plugins.job.prepost': [
+                  'jobscripts = avocado.plugins.jobscripts:JobScripts',
+                  ],
```

Comment:
Actually not just a symlink... there are additional features, so let's just attach some reference to move sysinfo hooks to this interface.